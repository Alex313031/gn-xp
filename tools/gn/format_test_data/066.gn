# Suppress sorting based on comment.

# NOSORT
sources = []

# NOSORT
sources = [
  "a",
]

# NOSORT
sources += [
  "a",
]

# NOSORT
# This is NOSORT because reason.
sources = [
  "z",
  "z2",
  "a",
  "y.cc",
]

# This is NOSORT because reason:
# NOSORT
sources += [
  "z",
  "z2",
  "a",
  "y.cc",
]

# This is SORT because reason:
# SORT
keys = [
  "z",
  "z2",
  "a",
  "y",
]

# This is SORTED as DEPS because reason:
# DEPSSORT
exclude_target = [
  "//z/a",
  ":z2",
  "//z/a:test",
  "y",
]
