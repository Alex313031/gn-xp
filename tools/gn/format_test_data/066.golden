# Suppress sorting based on comment.

# NOSORT
sources = []

# NOSORT
sources = [
  "a",
]

# NOSORT
sources += [ "a" ]

# NOSORT
# This is NOSORT because reason.
sources = [
  "z",
  "z2",
  "a",
  "y.cc",
]

# This is NOSORT because reason:
# NOSORT
sources += [
  "z",
  "z2",
  "a",
  "y.cc",
]

# This is SORT because reason:
# SORT
keys = [
  "a",
  "y",
  "z",
  "z2",
]

# This is SORTED as DEPS because reason:
# DEPSSORT
exclude_target = [
  ":z2",
  "y",
  "//z/a",
  "//z/a:test",
]
